### ISSUE TITLE
Display email address, not Globus username

### ISSUE THREAD

**Issue by shirey** [hubmapconsortium/ingest-ui#78]

**Comment by shirey** [comment]
@cborromeo @etlds  I still see some non-email-address-looking names 

![image](https://user-images.githubusercontent.com/5816400/74184394-b9e3ea80-4c14-11ea-84ba-bd4f456a8f9d.png)

**Comment by cborromeo** [comment]
@shirey I spoke to Desheng and you suggest we keep using the email address.  I extract the user's email address from their nexus token.  I extract the email address using this call:

`userinfo = authcache.getUserInfo(nexus_token, True)`

  This call returns:

`{'active': True,
 'aud': ['nexus.api.globus.org', '21f293b0-5fa5-4ee1-9e0e-3cf88bd70114'],
 'client_id': '21f293b0-5fa5-4ee1-9e0e-3cf88bd70114',
 'dependent_tokens_cache_id': 'bcc37f545b3b43653875ae5288a5d09e9868d72b0b45a8637627703bb9e0ed84',
 'email': 'chuck.hubmaptest@gmail.com',
 'exp': 1581608332,
 'hmgroupids': ['5777527e-ec11-11e8-ab41-0af86edb4424',
                '5bd084c8-edc2-11e8-802f-0e368f3075e8'],
 'hmroleids': [],
 'hmscopes': ['urn:globus:auth:scope:nexus.api.globus.org:groups'],
 'iat': 1581435532,
 'iss': 'https://auth.globus.org',
 'name': 'Chuck Borromeo',
 'nbf': 1581435532,
 'scope': 'urn:globus:auth:scope:nexus.api.globus.org:groups',
 'sub': 'a79606b3-e9be-4f1b-a01f-4aa1e8b900d8',
 'token_type': 'Bearer',
 'username': 'chuck.hubmaptest@gmail.com'}` 

Currently, I store the 'email' attribute from the userinfo object in Neo4j.  This is what appears on the front end.  Do you want me to use the 'username' attribute instead?  Is there some way we can confirm that the username field will solve this issue?

**Comment by etlds** [comment]
@cborromeo @shirey 
It used to store the "username" and I changed to "email" in this json. Some of the Havard guys has the long email address in the "username" field.

**Comment by shirey** [comment]
@cborromeo, yes- we need to store the email field, not username.  However, it looks like the production database will need to be updated because there are some stored usernames in there now.

**Comment by cborromeo** [comment]
@shirey @etlds I've got it.  I'll start looking through the test data for "bad" email addresses.

**Comment by cborromeo** [comment]
Here is the cypher I need to run on production:

MATCH (e) WHERE e.provenance_user_email in ['046f68566959fc5f01695e6e39ac002c@ucsd.edu', 'b1lake@eng.ucsd.edu'] SET e.provenance_user_email = 'b1lake@ucsd.edu' RETURN e
